.github/pull_request_template.md

```diff
@@ -28,6 +28,7 @@ Example: ```SUMMARY: Content "Adds new mutation category 'Mouse'"```
 
 See the [Changelog Guidelines](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md) for explanations of the categories.
 -->
+```SUMMARY: Content "changeme"```
```

Comment:
I would suggest instead just adding a line break after the colon on line 12, so that
```SUMMARY: [Category] "[description]"```
is a new line.